packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```
@@ -32,5 +32,7 @@
   "pci_database_add_create_database_error": "Une erreur est survenue lors de la création de votre service. Veuillez retenter l’opération ultérieurement : {{message}}",
   "pci_database_add_name_min_rule": "Au moins {{min}} caractères",
   "pci_database_add_name_max_rule": "Maximum {{max}} caractères",
-  "pci_database_add_name_pattern_rule": "Doit uniquement contenir des nombres, lettres, underscores, tirets ou points."
+  "pci_database_add_name_pattern_rule": "Doit uniquement contenir des nombres, lettres, underscores, tirets ou points.",
+  "pci_database_add_footer_part_1": "Tous les noms de produits, logos et marques sont la propriété de leurs propriétaires respectifs. Tous les noms d'entreprise, de produit et de service utilisés dans ce site Web sont à des fins d'identification seulement. L'utilisation de ces noms, logos et marques n'implique aucune approbation.",
+  "pci_database_add_footer_part_2": "*Redis est une marque de Redis Ltd. Tous les droits qui y sont attachés sont réservés à Redis Ltd. Toute utilisation par OVH SAS est uniquement à titre de référence et n'indique aucun sponsoring, soutien ou affiliation entre Redis et OVH SAS."
```

I think it must be **OVHcloud** instead of **OVH**

```suggestion
  "pci_database_add_footer_part_2": "*Redis est une marque de Redis Ltd. Tous les droits qui y sont attachés sont réservés à Redis Ltd. Toute utilisation par OVHcloud SAS est uniquement à titre de référence et n'indique aucun sponsoring, soutien ou affiliation entre Redis et OVHcloud SAS."
```

---

In french, it's OVH SAS https://www.ovhcloud.com/fr/terms-and-conditions/.
This depending the country.

---

Got it, thanks